samples/NovelRT.DotNet.Sample/Program.cs

```diff
@@ -0,0 +1,391 @@
+// Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root for more information.
+
+using System;
+using System.IO;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Text;
+using NovelRT.Interop;
+using static System.Net.Mime.MediaTypeNames;
+using static NovelRT.Interop.NovelRT;
+using static NovelRT.Interop.NrtBool;
+using static NovelRT.Interop.NrtResult;
```

Comment:
Not a complaint, just commenting how this confused me at first, It all makes sense now I see these two using statics ahahahahaha

Reply:
They just exist so you can do `NRT_SUCCESS` and not `NrtResult.NRT_SUCCESS`, makes it simpler to read and port the logic 😄

Reply:
yep figured that out after a few minutes of me going "wut?"lmao